build.gradle

```diff
@@ -53,7 +56,7 @@ buildscript {
     classpath 'com.vanniktech:gradle-android-junit-jacoco-plugin:0.13.0'
     classpath 'com.vanniktech:gradle-maven-publish-plugin:0.6.0'
 
-    classpath 'com.android.tools.build:gradle:3.1.2'
+    classpath 'com.android.tools.build:gradle:3.3.0-beta03'
```

Comment:
Can we use 3.2.1 for the time being? Or do we need 3.3.0 beta for something special?

Reply:
yup, it snuck in by accident. I'll use `3.2.1`